### PR TITLE
[UX] Clean up Vite boilerplate content - UX web app for marketing site and management portal

### PR DESCRIPTION
**Pull request by akiskips**
## 🛠️ Description
Removed some of the boilerplate code of the Vite React template.
Listed two commands to start the dev server and build the project on the Readme.md

## 📷 Screenshots
image

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [x] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [ ] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Updated changelog (required for `dev` PRs)
> - [x] ➡️ Will add log in a future PR (feature branch PRs only)
> - [ ] ❎ Log not needed (small/internal change)

### 📖 Did you update documentation?

> - [ ] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [x] ➡️ Will add docs in a future PR (feature branch PRs only)
> - [ ] ❎ Docs not needed (small/internal change)

### CODE REVIEW

**Comment by akiskips** [comment]
@microsoft-github-policy-service agree

File: src/web/index.html [hunk 1]
```diff
@@ -2,9 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <title>FinOps tookit Marketing Site</title>
```
Comment (flanakin) [inline src/web/index.html:6]:
```suggestion
    <title>FinOps toolkit</title>
```

File: src/web/pages/App.tsx [hunk 2]
```diff
@@ -1,29 +1,16 @@
...
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>count is {count}</button>
-        <p>
-          Edit <code>pages/App.tsx</code> and save to test HMR
+      <h1>FinOps toolkit </h1>
```
Comment (flanakin) [inline src/web/pages/App.tsx:7]: Not sure if this is the right way to do a subheading, but copying this from the current site. Feel free to tweak as appropriate. Just wanted to bring in real content.
```suggestion
      <head>
        <h1>Kick start your FinOps efforts</h1>
        <h2>Automate and extend the Microsoft Cloud with starter kits, scripts, and advanced solutions to accelerate your FinOps journey.</h2>
      </head>
```

File: src/web/pages/App.tsx [hunk 3]
```diff
@@ -1,29 +1,16 @@
...
+      <p>
+          Marketing Site for the FinOps Toolkit
```
Comment (flanakin) [inline src/web/pages/App.tsx:9]: More from the current site...
```suggestion
          The Microsoft FinOps toolkit is an open-source collection of learning resources and customizable tools to help you adopt and implement FinOps capabilities that automate and extend the Microsoft Cloud. This includes:
```

File: src/web/pages/App.tsx [hunk 4]
```diff
@@ -1,29 +1,16 @@
...
         </p>
-      </div>
-      <p className="read-the-docs">Click on the Vite and React logos to learn more</p>
+        <p className="read-the-docs">
+        Learn more at <a href="https://microsoft.github.io/finops-toolkit/">Official FinOps toolkit Site</a>
+      </p>
```
Comment (flanakin) [inline src/web/pages/App.tsx:13]: The rest of the intro on the current site...
```suggestion
      <ul>
        <li>Starter kits that help you get up and running.</li>
        <li>Automation scripts to streamline cost configuration and management at scale.</li>
        <li>Advanced solutions to facilitate building custom capabilities.</li>
        <li>Learning resources and best practices about how to implement FinOps.</li>
      </ul>
      <p>
        Read on to learn about the tools and roadmap for the FinOps toolkit or how to get involved.
      </p>
```

File: src/web/README.md [hunk 5]
```diff
@@ -1,5 +1,11 @@
 # React + TypeScript + Vite
 
+To start the development server, navigate to src/web folderand run:
+```npm install```
+and then
+```npm run dev```
```
Comment (flanakin) [inline src/web/README.md:6]:
```suggestion
To start the development server, navigate to **src/web** folder and run: `npm install` and then `npm run dev`.
```

File: src/web/README.md [hunk 6]
```diff
@@ -1,5 +1,11 @@
...
+
+----------------------------------------------------------------------------------------------------------------------------
```
Comment (flanakin) [inline src/web/README.md:8]:
```suggestion
---
```

**Review by flanakin — APPROVED** [review]

**Comment by flanakin** [comment]
@allcontributors Please add akiskips for code

File: src/web/eslint.config.js [hunk 7]
```diff
@@ -16,13 +17,17 @@ export default tseslint.config(
...
     },
     rules: {
       ...reactHooks.configs.recommended.rules,
       'react-refresh/only-export-components': [
         'warn',
         { allowConstantExport: true },
       ],
+          // Enable its recommended rules
+    ...react.configs.recommended.rules,
+    ...react.configs['jsx-runtime'].rules,
```
Comment (Ivanmtta) [inline src/web/eslint.config.js:30]: fix indentation

**Review by Ivanmtta — APPROVED** [review]

**Review by flanakin — APPROVED** [review]

File: src/web/README.md [hunk 8]
```diff
@@ -1,5 +1,8 @@
 # React + TypeScript + Vite
 
+To start the development server, navigate to **src/web** folder and run: `npm install` and then `npm run dev`.
+
+---
```
Comment (flanakin) [inline src/web/README.md:5]:
```suggestion
---

```

File: src/web/index.html [hunk 9]
```diff
@@ -2,9 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <title>FinOps toolkit</title>
```
Comment (flanakin) [inline src/web/index.html:6]:
```suggestion
    <title>FinOps toolkit - Kick start your FinOps efforts</title>
```

**Review by flanakin — APPROVED** [review]

**Pull request merged**